gprofiler/utils.py

```diff
@@ -261,6 +266,45 @@ def get_run_mode() -> str:
         return "local_python"
 
 
+def run_in_ns(nstypes: List[str], callaback: Callable[[], None], target_pid: int = 1) -> None:
```

Comment:
I see you draw your inspiration from Gwen Stefani 😄 
```suggestion
def run_in_ns(nstypes: List[str], callback: Callable[[], None], target_pid: int = 1) -> None:
```

Reply:
Never listened to Hollaback Girl

Reply:
Will do now

Reply:
Oh okay, ofc I have

Reply:
Done